cupy/linalg/eigenvalue.py

```diff
@@ -156,4 +156,7 @@ def eigvalsh(a, UPLO='L'):
 
     .. seealso:: :func:`numpy.linalg.eigvalsh`
     """
-    return _syevd(a, UPLO, False)[0]
+    if a.ndim == 3:
+        return cusolver._syevj_batched(a, UPLO, False)[0]
```

Comment:
This part could also be:
```suggestion
    if a.ndim > 2:
        *batch_shape, m, n = a.shape
        w, _ = cusolver._syevj_batched(a.reshape(-1, m, n), UPLO, False)
        return w.reshape(*batch_shape, m)
```